core-bundle/src/Controller/FrontendModule/TwoFactorController.php

```diff
@@ -84,6 +89,10 @@ protected function getResponse(Template $template, ModuleModel $model, Request $
             return new Response('', Response::HTTP_NO_CONTENT);
         }
 
+        if (!$this->get('security.helper')->isGranted('IS_AUTHENTICATED_FULLY')) {
+            return new Response('', Response::HTTP_NO_CONTENT);
```

Comment:
Shouldn't we throw a `InsufficientAuthenticationException` exception in this case?

Reply:
Yeah why not.